odoo/tools/misc.py

```diff
@@ -1238,6 +1239,26 @@ def format_datetime(env, value, tz=False, dt_format='medium', lang_code=False):
     return babel.dates.format_datetime(localized_datetime, dt_format, locale=locale)
 
 
+def format_time(env, value, time_format='medium', lang_code=False):
```

Comment:
introduced in master at https://github.com/odoo/odoo/commit/01c279ced14e9ad23ce191e8c51abffb53d603c6 (as we gonna need it in planning).
You can rebase ;)